internal/graph/dotgraph_test.go

```diff
@@ -299,32 +299,32 @@ func TestTagCollapse(t *testing.T) {
 	}
 
 	tagSource := []*Tag{
-		makeTag("12mb", "mb", 12, 100, 100),
-		makeTag("1kb", "kb", 1, 1, 1),
-		makeTag("1mb", "mb", 1, 1000, 1000),
-		makeTag("2048mb", "mb", 2048, 1000, 1000),
-		makeTag("1b", "b", 1, 100, 100),
-		makeTag("2b", "b", 2, 100, 100),
-		makeTag("7b", "b", 7, 100, 100),
+		makeTag("12MB", "mb", 12, 100, 100),
```

Comment:
It is strange that you had to rename the inputs. I would expect that we still re-format "12mb" as "12MB" so that the output is consistent.